.gitignore

```diff
@@ -2,4 +2,18 @@
 target
 *.swp
 __pycache__
-.DS_Store
\ No newline at end of file
+# Mac crap 
+.DS_Store
+
+# Added after calibration notbeooks 
+.pytest_cache/
+/calibration/SPC_data
+
+# jupyter notebook stuff
+*.ipynb_checkpoints/
+
+.idea
+
+#ABC files
+*.pkl
+*.db
```

Comment:
It looks like you added this entry after `abc-manchester.db` was already added to git. Can I `git rm` it? The intention is not check in these ABC db files, right?